public/src/main/java/com/revenuecat/purchases/models/GooglePurchaseInfo.kt

```diff
@@ -0,0 +1,29 @@
+package com.revenuecat.purchases.models
+
+import com.android.billingclient.api.ProductDetails
+import com.revenuecat.purchases.ProductType
+
+sealed class GooglePurchaseInfo: PurchaseInfo {
```

Comment:
This should also be internal I think.

Reply:
Had to rework some things to make this happen! `GoogleStoreProduct` (and some other Google models) were actually in the public module so I moved them into the Google feature specific one and now this was able to happen 😊